app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
I'm sick of dancing around completely busted patterns that are somehow still regarded as sacred.  It is funny because normally in this situation, legacy status quo folks will argue "we need a deprecation warning!".   But that reveals how messed up this situation is, since **you don't get deprecation warnings for private methods**!  IF this hack is important to somebody, it is on them to provide an implementation here that is not an abomination.  We move them in the right direction by removing it (and the rest).